### PR TITLE
Onnx upsample

### LINKED ISSUE #97: pytorch2onnx script run failed !
**Describe the bug**

**Reproduction**
1. What command or script did you run?
```
python tools/pytorch2onnx.py configs/ocrnet/ocrnet_hr18_512x1024_160k
_cityscapes.py --checkpoint checkpoints/ocrnet_hr18_512x1024_160k_cityscapes_20200602_191001-b9172d0c.pth --output-file onnx_models/ocrnet_hr18_512x1024_160k_cityscapes_20200602_191001.onnx --shape 512 1024 --verify
```
2. Did you make any modifications on the code or config? Did you understand what you have modified?  NO modifications
3. What dataset did you use?

**Environment**

1. Please run `python mmseg/utils/collect_env.py` to collect necessary environment infomation and paste it here.

```
sys.platform: linux
Python: 3.8.3 (default, May 19 2020, 18:47:26) [GCC 7.3.0]
CUDA available: True
CUDA_HOME: /usr/local/cuda-10.1
NVCC: Cuda compilation tools, release 10.1, V10.1.243
GPU 0: GeForce GTX 1080
GCC: gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
PyTorch: 1.5.0+cu101
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2019.0.5 Product Build 20190808 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.21.1 (Git Hash 7d2fd500bc78936d1d648ca713b901012f470dbc)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_37,code=compute_37
  - CuDNN 7.6.3
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_INTERNAL_THREADPOOL_IMPL -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.6.0
OpenCV: 4.4.0
MMCV: 1.0.5
MMSegmentation: 0.5.1+3e49d0a
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 10.1
```

2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]  conda
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**
If applicable, paste the error trackback here.
```
Traceback (most recent call last):
  File "tools/pytorch2onnx.py", line 188, in <module>
    num_classes = segmentor.decode_head.num_classes
  File "/home/apuser/miniconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 593, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'ModuleList' object has no attribute 'num_classes'
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

### PR DESCRIPTION
**Pull request by drcut**
As ONNX does not support using nn.Upsample, we have to use our customized Upsample Module which is a simple wrapper of F.interpolate.
Besides, pytorch2onnx.py does not support models with multiple decode heads (like ocrnet), we support these models in this version.

Fixed #97

### CODE REVIEW

**Comment by xvjiarui** [comment]
Fixed #97

File: mmseg/ops/wrappers.py [hunk 1]
```diff
@@ -22,4 +24,19 @@ def resize(input,
...
                         f'input size {(input_h, input_w)} is `x+1` and '
                         f'out size {(output_h, output_w)} is `nx+1`')
+    if isinstance(size, torch.Size):
+        size = tuple(int(x) for x in size)
     return F.interpolate(input, size, scale_factor, mode, align_corners)
+
+
+class Upsample(nn.Module):
+
+    def __init__(self, scale_factor, mode, align_corners):
```
Comment (xvjiarui) [inline mmseg/ops/wrappers.py:34]: We may also support `size`.

**Review by hellock — APPROVED** [review]

**Comment by dzyjjpy** [comment]
@drcut @hellock when convert onnx, I met another issue related.
I can convert hrnet's pytorch's model to onnx(tmp.onnx);  --verify shows that: outputs  between tmp.onnx and pytorch model  have no difference.
But I can't convert tmp.onnx to coreml。 It shows issue about resize op(upsample op)

**Comment by dzyjjpy** [comment]
[tmp.zip](https://github.com/open-mmlab/mmsegmentation/files/5485766/tmp.zip)
here is the converted onnx

**Pull request merged**